### PR TITLE
some changes i applied

### PR DESCRIPTION
**Pull request by Seliro**

### CODE REVIEW

**Comment by peonso** [comment]
I was applying some of those stuff also. Gonna apply one by one if you don't mind. Also point out some stuff.

File: server/data/actions/scripts/other/food.lua [hunk 1]
```diff
@@ -73,8 +75,12 @@ function onUse(player, item, fromPosition, target, toPosition)
 		player:sendTextMessage(MESSAGE_STATUS_SMALL, "You are full.")
 	else
 		player:feed(food[1] * 12)
-		player:say(food[2], TALKTYPE_MONSTER_SAY)
 		item:remove(1)
+		if enableSound then
+			player:say(food[2], TALKTYPE_MONSTER_SAY)
```
Comment (peonso) [inline server/data/actions/scripts/other/food.lua]: There was no sound at 7.4? If i remember correctly it uses TALKTYPE_SAY, no?

File: server/data/spells/scripts/attack/great energy beam.lua [hunk 2]
```diff
@@ -1,6 +1,6 @@
 local combat = createCombatObject()
 setCombatParam(combat, COMBAT_PARAM_TYPE, COMBAT_ENERGYDAMAGE)
-setCombatParam(combat, COMBAT_PARAM_EFFECT, CONST_ME_TELEPORT)
+setCombatParam(combat, COMBAT_PARAM_EFFECT, CONST_ME_EXPLOSIONHIT)
```
Comment (peonso) [inline server/data/spells/scripts/attack/great energy beam.lua]: Is this right?

**Comment by Seliro** [comment]
it's an option anyway, yeah feel free i really suck at this github thing..

File: server/data/spells/scripts/attack/great energy beam.lua (hunk 2 above)
Comment (Seliro) [inline server/data/spells/scripts/attack/great energy beam.lua]: yes

File: server/data/actions/scripts/other/food.lua (hunk 1 above)
Comment (Seliro) [inline server/data/actions/scripts/other/food.lua]: it's what i heard but i cant confirm, atleast we have an option for it now

Comment (peonso) [inline server/data/actions/scripts/other/food.lua]: You are right, I mixed up with 7.72.

**Comment by peonso** [comment]
I think it's all applied in those 3 commits, or was already applied before.
https://github.com/babymannen/theforgottenserver-7.4/commit/65303fc896d315eea5038b55ca679fdec36f2b4c
https://github.com/babymannen/theforgottenserver-7.4/commit/67856d5fea8a77bd0db5a6b6684d9274fb61bcbf
https://github.com/babymannen/theforgottenserver-7.4/commit/7d8df943efd946a06f77ca76ea07400e055c1832

**Comment by peonso** [comment]
I think it's all applied in those 3 commits, or was already applied before.
https://github.com/babymannen/theforgottenserver-7.4/commit/65303fc896d315eea5038b55ca679fdec36f2b4c
https://github.com/babymannen/theforgottenserver-7.4/commit/67856d5fea8a77bd0db5a6b6684d9274fb61bcbf
https://github.com/babymannen/theforgottenserver-7.4/commit/7d8df943efd946a06f77ca76ea07400e055c1832

**Pull request closed without merging**